helmfile.d/charts/prometheus-alerts/templates/alerts/cluster-capacity-management-alerts.yaml

```
@@ -137,4 +137,13 @@ spec:
       for: 5m
       labels:
         severity: warning
+    - alert: NodeMissingElastisysLabel
```

Suggestion: How about making it clearer that it is the `node-group` label that is missing in case we have other `elastisys` labels in the future?
```suggestion
    - alert: NodeMissingElastisysNodeGroupLabel
```

---

done with https://github.com/elastisys/compliantkubernetes-apps/commit/2e4aaf8d69b67dd9ce10fc20bb741ba7787fb6e4